recipes/sequenoscope/meta.yaml

```diff
@@ -17,10 +17,10 @@ build:
 
 requirements:
     build:
-        - python >=3.7.0
+        - python =3.7.12
         - pip >=23.2.1
     run:
-        - python >=3.7.0
+        - python =3.7.12
```

Comment:
Why not leave as `- python >=3.7` in run as it will be more easier for conda solver to adapt existing environment?